django/conf/global_settings.py

```diff
@@ -502,6 +502,17 @@
 CACHE_MIDDLEWARE_SECONDS = 600
 CACHE_MIDDLEWARE_ALIAS = 'default'
 
+#########
+# TASKS #
+#########
+
+# The task backends to use.
+QUEUES = {
```

Comment:
Actually, wouldn't these be more `TASK_BROKERS` than `QUEUES`? One broker again could have multiple queues again if talking in celery/rabbitmq terms.